discord/flags.py

```diff
@@ -1436,3 +1466,107 @@ class ChannelFlags(BaseFlags):
     def pinned(self):
         """:class:`bool`: Returns ``True`` if the thread is pinned to the forum channel."""
         return 1 << 1
+
+
+class ArrayFlags(BaseFlags):
+    @classmethod
+    def _from_value(cls: Type[Self], value: List[int]) -> Self:
+        self = cls.__new__(cls)
+        self._value = sum((1 << i - 1 for i in value))
+        return self
+
+    @property
+    def value(self) -> List[int]:
+        ret = []
+        for idx, (_, value) in enumerate(self, start=1):
+            if value:
+                ret.append(idx)
+
+        return ret
+
+
+@fill_with_flags()
+class AutoModPresets(ArrayFlags):
```

Comment:
Don't see this class mentioned anywhere in api.rst
Also flags (`profanity`, `sexual_content`, `slurs`) should probably have descriptions?

Reply:
Good catch, thanks